content/source/docs/cloud/api/variable-sets.html.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: "cloud"
+page_title: "Variable Sets - API Docs - Terraform Cloud and Terraform Enterprise"
+---
+
+# Variable Sets API
+
+This set of APIs covers create, update, list, delete, and some specialized operations.
+
+Viewing Variable Sets requires access to the owning organization. Destructive actions require org admin status or workspace management permissions.
+
+
+## Create a Variable Set
+`POST organizations/:organization_name/varsets`
+
+Example Payload:
+```json
+{:data=>
+  {
+    :type=>"varsets",
+    :attributes=>{
+      :name=>"varset-7f7b6783",
+      :description=>"Full of vars and such for mass reuse",
+      :is_global=>false
+    },
+    :relationships=>{
+      :workspaces=>{
+        !! Sending an empty workspaces list is an explicit declaration to unassign from all workspaces
+        :data=>[
+          {
+            :id=>"ws-z6YvbWEYoE168kpq",
+            :type=>"workspaces"
+          },
+          ...
+        ]
+      },
+      :vars=>{
+        :data=>[
+          {
+            :type=>"vars",
+            :attributes=>{
+              :key=>"c2e4612d993c18e42ef30405ea7d0e9ae",
+              :value=>"8676328808c5bf56ac5c8c0def3b7071",
+              :category=>"terraform"
+            }
+          },
+          ...
+        ]
+      }
+    }
+  }
+}
+```
+
+rsponds with JSON representation of the save Variable Set (see list Variable Set section for example)
+
+## Update a Variable Set
+`PATCH varsets/:external_id`
+
+rsponds with JSON representation of the save Variable Set (see list Variable Set section for example)
+
+## Delete a Variable Set
+`PUT/PATCH varsets/:external_id`
+
+on success, responds with no content
+
+## List Variable Set
+`GET organizations/:organization_name/varsets`
```

Comment:
Can we differentiate these endpoints in the documentation? Should workspaces be under workspaces? Organizations under organizations?